# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1315088938 14400
#      Sat Sep 03 18:28:58 2011 -0400
# Node ID 87015276d6255274954d630beb30343588462eb4
# Parent  4ffea87ad71b3425970d383e89359fde9820a897
codesprint: demos for title

* title.m: New demos.

diff --git a/scripts/plot/title.m b/scripts/plot/title.m
--- a/scripts/plot/title.m
+++ b/scripts/plot/title.m
@@ -32,8 +32,25 @@ function h = title (string, varargin)
     else
       __axis_label__ ("title", string, varargin{:});
     endif
   else
     print_usage ();
   endif
 
 endfunction
+
+%!demo
+%! clf ();
+%! ax=axes();
+%! xl = get(ax,"title");
+%! title("Testing title")
+%! assert(get(xl,"string"),"Testing title")
+
+%!demo
+%! clf ();
+%! plot3 ([0,1], [0,1], [0,1]);
+%! xl = get(gca (), "title");
+%! title("Testing title")
+%! assert(get(xl,"string"),"Testing title")
+
+## Remove from test statistics.  No real tests possible.
+%!assert (1)
